src/components/Footer.js

```diff
@@ -1,72 +1,94 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect, Suspense } from 'react'
+import useLanguage from '../hooks/useLanguage'
 
-const quotes = [
-  {
-    id: 1,
-    text:
-      'Education is the most powerful weapon which you can use to change the world.',
-    author: 'Nelson Mandela',
-  },
-  {
-    id: 2,
-    text:
-      'Education is the passport to the future, for tomorrow belongs to those who prepare for it today.',
-    author: 'Malcolm X',
-  },
-  {
-    id: 3,
-    text:
-      'If you talk to a person in a language they understand, that goes to their head. If you talk to them in their own language, that goes to their heart.',
-    author: 'Nelson Mandela',
-  },
-]
+const quotes = {
+  en: [
+    {
+      text:
+        'Education is the most powerful weapon which you can use to change the world.',
+      author: 'Nelson Mandela',
+    },
+    {
+      text:
+        'Education is the passport to the future, for tomorrow belongs to those who prepare for it today.',
+      author: 'Malcolm X',
+    },
+    {
+      text:
+        'If you talk to a person in a language he understands, that goes to his head. If you talk to them in their own language, that goes to their heart.',
+      author: 'Nelson Mandela',
+    },
+  ],
+  sw: [
+    {
+      text: 'Elimu ni silaha kubwa ambayo unaweza kuitumia kubadili dunia.',
+      author: 'Nelson Mandela',
+    },
+    {
+      text:
+        'Elimu ni pasipoti kwa siku zijazo, maana kesho ni kwa ajili ya wale wanaoiandaa leo hii.',
+      author: 'Malcom X',
+    },
+    {
+      text:
+        'Kama ukiongea na mtu kwa lugha ambayo anaielewa basi itakaa kichwani. Ila ukiongea nao kwa lugha ya kwao basi itakaa moyoni.',
+      author: 'Nelson Mandela',
+    },
+  ],
+}
 
 const Footer = () => {
   const [randomQuote, setRandomQuote] = useState(null)
+  const { language } = useLanguage()
+  const quotesInLanguage = quotes[language]
+  console.log(language)
```

Comment:
Nice work Rafia. 
One minor change here can you please add a temp condition here or a try and catch for this line cause if we change the language to `Amharic` or `Arabic` the website will break as we do not have the quotes localized for those languages yet. 
Something like 
```
const quoteLanguage = language === 'am' || language === 'ar' ? 'en' : language;
const quotesInLanguage = quotes[quoteLanguage]
```

Can you remove the console log please as well?

Reply:
Thanks Khalid.
Done.